geode-core/src/main/java/org/apache/geode/management/internal/configuration/messages/ConfigurationResponse.java

```diff
@@ -144,6 +144,6 @@ public void setMember(DistributedMember member) {
 
   @Override
   public Version[] getSerializationVersions() {
-    return new Version[] {Version.CURRENT};
+    return null;
```

Comment:
How does this change relate to the rest of this PR?

Reply:
I ran into a failure caused by this code in testing rolling-upgrade code where I purposefully introduced an older version into a cluster already in the process of upgrading.

Reply:
This is GEODE-6956, so I'll link that ticket to this one.